src/main/java/com/example/demo/model/NotTrueFalseValidator.java

```diff
@@ -0,0 +1,31 @@
+package com.example.demo.model;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.example.demo.Dao.DaoImpl.StaffDaoImpl;
+
+public class NotTrueFalseValidator implements ConstraintValidator<NotTrueFalse, String>  {
+	
+	@Autowired
+    StaffDaoImpl staffDaoImpl;
+	
+	@Override
+    public void initialize(NotTrueFalse constraintAnnotation) {
+    }
+	
+	@Override
+	public boolean isValid(String newGladFlgValue, ConstraintValidatorContext context) {
+		
+		User user = staffDaoImpl.findNewGladFlg(newGladFlgValue);
+		if(user!=null) {
+            return false;
+        }
+        
+        return true;
+		
+	}
```

Comment:
```suggestion
public boolean isValid(String newGladFlgValue, ConstraintValidatorContext context) {
    User user = staffDaoImpl.findNewGladFlg(newGladFlgValue);
    if(user != null) {
        return false;
    }
    return true;
}
```
インデントが全体的にズレてます。
インデントのズレはPHPでは処理に影響ないですが、言語によっては処理もバグります。
それに例え処理がバグらなくても可読性は落ち人的バグの要因になるので直しておいてください。

また、ここのインデント整形しようとして気づいたんですが、インデントにタブとスペースが混在しています。
どちらを使うかはプロジェクトによる部分もありますが
今回はインデントについてはすべてスペース（1インデントについては4半角スペース）で統一して欲しいです。

Reply:
インデントに関して、いつもオートフォーマットを使用しているのですが、
なんとなき横に長くなってきたからということでやってしまうこともあるため、
これからは可読性について、考えながら、フォーマットの訂正をしていきます。
また、このファイルはカスタムバリデーションに関するファイルのため、
不要になったため、削除いたしました。